{"name": "test01_update_success", "status": "failed", "statusDetails": {"message": "AssertionError: assert '成功' in '{\"msg\":\"操作失败\",\"code\":500}'\n +  where '{\"msg\":\"操作失败\",\"code\":500}' = <Response [200]>.text", "trace": "self = <KDTX.script.test08_update_course.TestCourseAPI object at 0x00000196AA71B2E0>\n\n    def test01_update_success(self):\n        update_data = {\n            \"id\": 100,\n            \"name\": \"接口测试001\",\n            \"subject\": \"6\",\n            \"price\": 998,\n            \"applicablePerson\": \"2\",\n            \"info\": \"课程介绍001\",\n        }\n        response = self.course_api.update_course(test_data=update_data, token=TestCourseAPI.TOKEN)\n        print(response.json())\n        # 断言响应状态码\n        assert 200 == response.status_code\n        # 断言返回消息\n>       assert \"成功\" in response.text\nE       assert '成功' in '{\"msg\":\"操作失败\",\"code\":500}'\nE        +  where '{\"msg\":\"操作失败\",\"code\":500}' = <Response [200]>.text\n\nscript\\test08_update_course.py:46: AssertionError"}, "start": 1725005271840, "stop": 1725005271997, "uuid": "3fe845f1-ba2f-4657-bef4-c8332b408384", "historyId": "3f2bf7e7755ef2fe9aa4733275a98ad5", "testCaseId": "3f2bf7e7755ef2fe9aa4733275a98ad5", "fullName": "script.test08_update_course.TestCourseAPI#test01_update_success", "labels": [{"name": "parentSuite", "value": "script"}, {"name": "suite", "value": "test08_update_course"}, {"name": "subSuite", "value": "TestCourseAPI"}, {"name": "host", "value": "china-net"}, {"name": "thread", "value": "6252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "script.test08_update_course"}]}